ui/fancy/src/css/style.less

```diff
@@ -0,0 +1,149 @@
+@color-background: #622870;
+@color-background-grid: #2d0238;
+@color-block-red: #a7383e;
+@color-block-green: #2c8437;
+@color-block-blue: #303e73;
+@color-swapper: #ffffff;
+
+@grid-width: 6;
+@grid-height: 8;
+
+@block-size: 40px;
```

Comment:
I think we should measure things in ems as we are populating the blocks with text content.